docker/database-setup/16-add-log-template-layout-column.sql

```diff
@@ -0,0 +1,2 @@
+alter table log_templates 
+add column layout jsonb default '[]'::jsonb;
```

Comment:
I'm defaulting this to an array, but the shared type expects a record. I don't think it really matters because we're probably always going to pass a `layout`, but set the default to `{}` just in case.